src/lib/y2network/autoinst/interfaces_reader.rb

```diff
@@ -47,7 +46,10 @@ def config
           log.info "Creating config for interface section: #{interface_section.inspect}"
           config = create_config(interface_section)
           config.propose # propose reasonable defaults for not set attributes
-          load_generic(config, interface_section)
+          unless load_generic(config, interface_section)
+            log.info "Skipping interface as the configuration was wrongly defined"
```

Comment:
I think we should warn the user (using autoinstallation issues, for instance).

Reply:
Yep, that is the idea as commented in the TODO but I planned to add it only in master.